public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -123,7 +130,7 @@ export class SharedPreferences extends PureComponent<Props, State> {
           <Select
             value={dashboards.find(dashboard => dashboard.id === homeDashboardId)}
             getOptionValue={i => i.id}
-            getOptionLabel={i => i.title}
+            getOptionLabel={(dashboard: DashboardSearchHit) => this.getFullDashName(dashboard)}
```

Comment:
you do not need to create a function here every render, just do getOption={this.getFullDashName}